{
    "checksum": "aeff0dd7aa02673e6d99ad56abef6019a22630015a5e3a581a9a4facfa4d7c62",
    "file": "test_table.test_create_table_with_already_existing_table_fail_on_exist.yaml",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/plain"
    },
    "number_of_duplicates": 1,
    "package": "azure-cosmos-table-python",
    "path": "azure-cosmos-table-python/1.0.5+git20191025-5/azure-cosmosdb-table/tests/recordings/test_table.test_create_table_with_already_existing_table_fail_on_exist.yaml",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/azure-cosmos-table-python/1.0.5+git20191025-5/",
        "metric": {
            "size": 6140
        },
        "pts_link": "https://tracker.debian.org/pkg/azure-cosmos-table-python",
        "sloc": [
            [
                "python",
                8856
            ],
            [
                "makefile",
                265
            ],
            [
                "sh",
                7
            ]
        ],
        "suites": [
            "bookworm",
            "bullseye",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/python-team/modules/azure-cosmos-table-python",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/a/azure-cosmos-table-python/1.0.5%2Bgit20191025-5/azure-cosmosdb-table/tests/recordings/test_table.test_create_table_with_already_existing_table_fail_on_exist.yaml",
    "stat": {
        "perms": "rw-r--r--",
        "size": 2684,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "1.0.5+git20191025-5"
}